src/apis/loanProducts.ts

```diff
@@ -0,0 +1,33 @@
+import axios from 'axios';
+
+const API_URL = process.env.API_URL || 'http://localhost:8080';
+
+/**
+ * 대출 상품 목록 전체 조회 API
+ * @return data
+ */
+const getAllProducts = async () => {
+    const {data} = await axios.get(`${API_URL}/api/loans`);
```

Comment:
제가 apiClient를 만들어놔서 

다음처럼 바꾸시면 될 것 같아요~

```suggestion
      const { data } = await apiClient.get(`/api/loans`);
```